### ISSUE TITLE
Modrinth often fails to initialize

### ISSUE THREAD

**Issue by nicecraftz** [modrinth/code#2864]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [x] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [x] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

v0.8.9

### What operating systems are you seeing the problem on?

Windows

### Describe the bug

When i try to load modrinth launcher from windows search bar it always fails to initialize, if i use the desktop shortcut it works fine, but if i try to add the app to taskbar it fails again to initialize.

I tried reinstalling 3 times but nothing worked 👎🏻 

https://github.com/user-attachments/assets/402f24b5-dbdc-4fe6-98a8-69feea86a07b

### Expected behavior

Loading

**Comment by Minenash** [comment]
Do both shortcuts go to the same location?

**Comment by nicecraftz** [comment]
No, a folder called "Modrinth App" instead of "ModrinthApp" existed. i deleted the other one

**Comment by Algiuxs** [comment]
> No, a folder called "Modrinth App" instead of "ModrinthApp" existed. i deleted the other one

uninstall modrinth, then delete all modrinth folders, then install modrinth.
